site/tags/messages.tag

```diff
@@ -30,25 +30,30 @@
         return
       }
 
-      const { messages } = await get(serverURL, `channel/${channelID}/latest-messages`)
+      let fetchedServerURL
+      const { messages } = await get(fetchedServerURL = serverURL, `channel/${channelID}/latest-messages`)
       const newMessageGroups = messages.reduce((groups, msg) => {
         const lastGroup = groups[groups.length - 1]
 
         if (shouldAppendMsgToGroup(msg, lastGroup)) {
           lastGroup.messages.push(msg)
+          return groups
         } else {
-          groups.push({
+          return [ ...groups, {
             authorID: msg.authorID,
             authorUsername: msg.authorUsername,
             channelID: msg.channelID,
             date: msg.date,
             messages: [ msg ],
-          })
+          } ]
```

Comment:
Sure, this works.

I'm not sure I like the code style though :upside_down_face: